MIDAS/src/sensors.h

```diff
@@ -26,6 +26,10 @@ struct BarometerSensor {
 struct ContinuitySensor {
     ErrorCode init();
     Continuity read();
+
+    // IIRC we have 6 continuity pins
+    static int const PIN_COUNT = 6;
+    uint16_t pins[PIN_COUNT];
```

Comment:
why do we have the pins array in the struct for the sensor as well as the struct for the sensor data? I don't think this array is necessary

Reply:
I think we like #defines more than static consts.